apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.DatabaseStatistics.cs

```diff
@@ -111,7 +103,7 @@ public override string ToString()
                 string status = Shutdown ? "Shutdown"
                     : Healthy ? "Healthy"
                     : "Unhealthy";
-                return $"Database: {DatabaseName}; Active: {ActiveSessionCount}; Read pool: {ReadPoolCount}; Write pool: {ReadWritePoolCount}; In-flight creation: {InFlightCreationCount}; Pending acquisitions: {PendingAcquisitionCount}; Status: {status}; Read/write prewarming: {ReadWriteTransactionRequestsPrewarmed}/{ReadWriteTransactionRequests}";
+                return $"{PoolSegmentStatistics}; Active: {ActiveSessionCount}; Read pool: {ReadPoolCount}; Write pool: {ReadWritePoolCount}; In-flight creation: {InFlightCreationCount}; Pending acquisitions: {PendingAcquisitionCount}; Status: {status}; Read/write prewarming: {ReadWriteTransactionRequestsPrewarmed}/{ReadWriteTransactionRequests}";
```

Comment:
The "{PoolSegmentStatistics" part is going to give the same information twice.

Can't we just have `public override string ToString() => PoolSegmentStatistics.ToString();`?